### DISCUSSION TITLE
Is there any possible way to control Move Hub (Lego Boost) via PC keyboard?

### DISCUSSION THREAD

**Discussion by Slasher111** [pybricks/support#529] [Q&A]
I've already understood that it's impossible to use the "Keyboard Remote Control" from the LEGO Technic examples because of the absense of 'usys' and 'uselect' modules in Move Hub micropython version. So is there any tiny chance to do it another way? Thank you in advance even if the answer is "no" :)

**Comment by dlech** [comment]
I think we had to disable all of the functions that could do this due to the small flash memory size on the Move hub. For example, there is an `input()` function that could work (blocking and requires pressing enter), but I think that is disabled too.
